src/kv/test/null_encryptor.h

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the Apache 2.0 License.
+#pragma once
+
+#include "crypto/symmetric_key.h"
+#include "kv/kv_types.h"
+
+namespace kv
+{
+  // NullTxEncryptor does not decrypt or verify integrity
+  class NullTxEncryptor : public kv::AbstractTxEncryptor
+  {
+  public:
+    void encrypt(
+      const std::vector<uint8_t>& plain,
+      const std::vector<uint8_t>& additional_data,
+      std::vector<uint8_t>& serialised_header,
+      std::vector<uint8_t>& cipher,
+      kv::Version version) override
+    {
+      crypto::GcmHeader<crypto::GCM_SIZE_IV> gcm_hdr = {};
+      serialised_header = std::move(gcm_hdr.serialise());
```

Comment:
Minor: I'm surprised that the null encryptor needs a correctly-sized-but-invalid header, but haven't tried removing it yet to see what breaks.

Reply:
Good point. I can't remember either. It looks like it's been there since the beginning of time so it might not break anything.

Reply:
Looks like this is safe to remove, so I've removed it.